Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeView.Properties.cs

```diff
@@ -27,12 +29,66 @@ public partial class BladeView
         public static readonly DependencyProperty ActiveBladesProperty = DependencyProperty.Register(nameof(ActiveBlades), typeof(IList<BladeItem>), typeof(BladeView), new PropertyMetadata(null));
 
         /// <summary>
+        /// Identifies the <see cref="FullScreenBlades"/> attached property.
+        /// </summary>
+        public static readonly DependencyProperty FullScreenBladesProperty = DependencyProperty.RegisterAttached(nameof(FullScreenBlades), typeof(bool), typeof(BladeView), new PropertyMetadata(false, FullScreenBladesChangedCallback));
+
+        /// <summary>
         /// Gets or sets a collection of visible blades
         /// </summary>
         public IList<BladeItem> ActiveBlades
         {
             get { return (IList<BladeItem>)GetValue(ActiveBladesProperty); }
             set { SetValue(ActiveBladesProperty, value); }
         }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether blades are full screen
+        /// </summary>
+        public bool FullScreenBlades
```

Comment:
Change this to an enum to not limit to only two values. Perhaps 'BladeMode' with values 'Normal' and 'Full'